browser/src/map/handler/Map.Keyboard.js

```diff
@@ -469,6 +469,10 @@ window.L.Map.Keyboard = window.L.Handler.extend({
 			}
 			ev.preventDefault();
 		}
+		else if (this._isCtrlKey(ev) && ev.keyCode === this.keyCodes.P) {
```

Comment:
please use new approach for key shortcuts: https://github.com/CollaboraOnline/online/blob/master/browser/src/global.d.ts#L245